azure/virtual-machine/examples/advanced/README.md

```diff
@@ -0,0 +1,43 @@
+# Example - Advanced
+
+This example is used to illustrate the advanced usage of this module by using more variables and implementing availability sets, load balancing and NSGs.
+
+> NOTE: The values for `backend_address_pool_ids` depends on vales not known initially. These must be pre-applied as per below. Run the below to get it started, then a plain terraform apply.
+>
+> ```bash
+> terraform apply -target azurerm_lb_backend_address_pool.vm
```

Comment:
If we're having to work around like this, is it worth including this in the advanced example at all?

Reply:
Added into the kanban board for a future idea https://github.com/orgs/ans-coe/projects/1?pane=issue&itemId=34171353